pylint/checkers/imports.py

```diff
@@ -712,6 +717,24 @@ def _check_deprecated_module(self, node, mod_path):
             if mod_path == mod_name or mod_path.startswith(mod_name + '.'):
                 self.add_message('deprecated-module', node=node, args=mod_path)
 
+    def _check_import_as_rename(self, node):
+        names = node.names
+        if not all(names[0]):
+            return
+
+        real_name = names[0][0]
+        head, tail = os.path.splitext(real_name)
+        if tail == '':
+            real_name = head
+        else:
+            real_name = tail[1:]
+        imported_name = names[0][1]
+        if real_name == imported_name:
+            msg = "'%s' is renamed as same package '%s', rename it to different one."
```

Comment:
Could this be reworded? `real_name` and `imported_name` will never be different in this message. [flake8-tidy-imports](https://github.com/adamchainz/flake8-tidy-imports) goes as far as offering the correct way to write the import. Alternatively we could say something like "Import alias does not rename '%s' and can be rewritten."?

Reply:
will refactor this.